build.gradle

```diff
@@ -0,0 +1,70 @@
+/**
+* The application plugin facilitates creating an executable JVM application.
+*/
+
+apply plugin: 'application'
+
+/**
+* This line specifies where the main.java file is found.
+*/
+
+mainClassName='View.Main'
+
+/**
+* This statement declares where Gradle is to find directories 
+* for source code, external resources, and test classes
+*/
+
+sourceSets {
+    main {
+		java {
+			srcDirs = ['src']
+		}
+		resources {
+			srcDirs = ['src']
+		}
+	}
+	
```

Comment:
This trailing white space needs to be removed.

Reply:
Deleted